src/ILCompiler.ReadyToRun/src/ObjectWriter/MetadataRvaFixupBuilder.cs

```diff
@@ -24,7 +25,7 @@ public class MetadataRvaFixupBuilder
         /// <param name="peReader">Input MSIL image reader</param>
         /// <param name="sectionRvaDelta">The difference in bytes between the metadata PE section 
```

Comment:
Nit - could you please update the xml comment?

Reply:
Done